front/src/api/common.ts

```diff
@@ -2,7 +2,7 @@
 import axios from "axios";
 import { z } from "zod";
 
-const HOST = "http://localhost:5000";
+const HOST = "";
```

Comment:
I think we should keep this default here to make development easier. We could also
read this value from an environment variable or configuration file so that
it is configurable at build time.